Jenkinsfile

```diff
@@ -104,25 +87,14 @@ pipeline {
 		stage('Post: Archive Outputs'){
 			steps{
 				script{
-					def s3Path = "${env.S3_RELEASE_DIRECTORY_URL}/${env.RELEASE_VERSION}/go_update"
-					sh "mkdir -p databases/ data/"
-					sh "mv --backup=numbered *_${env.RELEASE_VERSION}_*.dump.gz databases/"
-					sh "mv src/main/resources/go.obo data/"
-					sh "mv src/main/resources/ec2go data/"
-					sh "gzip -r data/* logs/*"
-					sh "mv go-update-v${env.RELEASE_VERSION}-reports.tgz data/"
-					sh "aws s3 --no-progress --recursive cp databases/ $s3Path/databases/"
-					sh "aws s3 --no-progress --recursive cp logs/ $s3Path/logs/"
-					sh "aws s3 --no-progress --recursive cp data/ $s3Path/data/"
-					sh "rm -r databases logs data reports"
+					def releaseVersion = utils.getReleaseVersion()
+					def dataFiles = ["src/main/resources/go.obo", "src/main/resources/ec2go", "go-update-v${releaseVersion}-reports.tgz"]
+					// GO Update log files are already in a folder called 'logs', so an empty list is passed.
+					def logFiles = []
+					utils.cleanUpAndArchiveBuildFiles("go_update", dataFiles, logFiles)
```

Comment:
Just wondering, has the change to archiving been tested?

Reply:
Archiving in this manner was used during the v75 release

Reply:
Yes, it was used during v75